src/components/Widgets/Widget.tsx

```diff
@@ -1,7 +1,6 @@
 'use client';
 import { ClientOnly } from '@/components/ClientOnly';
 import { MultisigWalletHeaderAlert } from '@/components/MultisigWalletHeaderAlert';
-import { widgetConfig } from '@/config/widgetConfig';
```

Comment:
You cannot do that otherwise the widget won't be initialized anymore

Reply:
Is that the case? Here, we have just imported the `widgetConfig`. The widgetConfig only included a set of tokens and the problem with that was, that tokens are being overwritten again in the Widget.tsx.

If "not importing" the widgetConfig does stop the initialization of the widget, we must find a solution. But it seemed to work just fine.